src/components/quiz/QuizQuestions.scss

```
@@ -0,0 +1,60 @@
+@import "../../constants.scss";
+
+*{  box-sizing: border-box;
+  background-color:$secondary-light;
+}  
+
+h1{
+  color: black;
+  font-family: 'Courier New', Courier, monospace;
+  font-weight: bold;
+  text-align: center;
+}
+h3{
```

Spaces between CSS blocks (for all of them in this file)
```suggestion
}

h3 {
```